lib/robots/dor_repo/ocr/start_ocr.rb

```diff
@@ -11,14 +11,29 @@ def initialize
 
         def perform_work
           if Dor::TextExtraction::Ocr.new(cocina_object:).possible?
-            object_client.version.open(description: 'Start OCR workflow') unless object_client.version.status.open?
+            open_object unless object_client.version.status.open?
           else
             # skip all steps in the WF with note
             note = 'No files available or invalid object for OCR'
             workflow_service.skip_all(druid:, workflow: 'ocrWF', note:)
             LyberCore::ReturnState.new(status: 'skipped', note:)
           end
         end
+
+        private
+
+        def open_object
+          tries = 0
+          begin
+            object_client.version.open(description: 'Start OCR workflow')
+          rescue Dor::Services::Client::Error => e
+            tries += 1
+            raise e unless tries < 3
+
+            Honeybadger.notify("[NOTE] Problem opening object version at start of ocrWF for #{druid}", context: { druid:, tries:, error: e })
```

Comment:
i think we might get slightly better notifications if the `for #{druid}` is removed from the notification text and left only in the `context` hash, since that should allow for easier error message collation by honeybadger.  i think it might still do the right thing, but it might also lead to totally separate notifications for each druid, instead of one encompassing notification bucket with a bunch of individual instances.

Reply:
good call, just removed druid from the text of the alert